var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -234,7 +234,7 @@
 
 @IntelOneApiPackage.update_description
 class IntelOneapiCompilers(IntelOneApiPackage, CompilerPackage):
-    """Intel oneAPI Compilers. Includes: icc, icpc, ifort, icx, icpx, and ifx."""
+    """Intel oneAPI Compilers. Includes: icx, icpx, ifx (in recent releases)."""
```

Comment:
```suggestion
    """Intel oneAPI Compilers. Includes: icx, icpx, ifx, and ifort. Releases before 2024.0 include icc/icpc"""
```

ifort is still there, but is likely to be removed soon.